.github/workflows/cd_release.yml

```diff
@@ -32,5 +33,24 @@ jobs:
       changelog_exclude_labels: "skip_changelog,duplicate,question,invalid,wontfix"
       docs_framework: mkdocs
     secrets:
-      PyPI_token: ${{ secrets.PYPI_TOKEN }}
+      #PyPI_token: ${{ secrets.PYPI_TOKEN }}
       PAT: ${{ secrets.RELEASE_PAT }}
+
+  publish:
+    name: Publish on PyPI
+    needs: build
+    runs-on: ubuntu-latest
+
+    #environment: release
+    permissions:
+      id-token: write
+
+    steps:
+      - name: Download built distritbution
+        uses: actions/download-artifact@v4
+        with:
+          name: dist
```

Comment:
Is there only one artifact that shuld be uploaded? Are you sure that  this is the correct name? Better to remove this line, to be sure, I think. 
```suggestion
```